must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -259,6 +259,7 @@ for ns in $namespaces; do
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log;
                     printf "Collecting snap info for: %s/%s\n" "${bp}" "${image}";
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log; 
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all --format=json --pretty-format $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log;
```

Comment:
Just writing "--format json-pretty" should work instead of "--format=json --pretty-format"
Also, the debug log file name should have "json" as suffix for better readability e.g. gather-rbd-snap-"${image}"-json-debug.log

Reply:
> Just writing "--format json-pretty" should work instead of "--format=json --pretty-format"

No, that doesn't work.

> Also, the debug log file name should have "json" as suffix for better readability e.g. gather-rbd-snap-"${image}"-json-debug.log

Done.